extension/chrome/elements/compose.ts

```diff
@@ -68,7 +68,19 @@ Catch.try(async () => {
       }
       return result;
     };
-    return storage.sendAs || (storage.addresses && arrayToSendAs(storage.addresses));
+    const result = storage.sendAs || (storage.addresses && arrayToSendAs(storage.addresses));
+    if (result && storage.email_footer) {
```

Comment:
I would ignore whatever was set in `email_footer`, because going forward, footers should just be set from Gmail.

If user has no footer set on Gmail, then there is no footer.

Reply:
Agree, removed